app/controllers/homepage_controller.rb

```diff
@@ -1,7 +1,4 @@
 class HomepageController < ApplicationController
-  def greeting
-    @message = "Write 3 or more things on your mind now."
-  end
 
   def index
     @thoughts = Thought.all
```

Comment:
This makes it possible to display thoughts on the home page in the view: `index.html.erb `